### PR TITLE
chore: add console e2e testing with playwright

### LINKED ISSUE #2416: Add Playwright tests to console.
We should try to use [Playwright](https://playwright.dev/) to run tests against a running ftl dev command

I think this can be do manually by running `npx playwright test` while `ftl dev` is running.
We can also make this an integration test which fires up and deploys a couple modules (probably `time` and `echo` for now) then run playwright as part of another integration test action.

### PR DESCRIPTION
**Pull request by wesbillman**
Fixes #2416

### CODE REVIEW

File: frontend/tests/verb.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  await expect(page.getByText('Body', { exact: true })).toBeVisible()
+  await expect(page.getByText('Verb Schema', { exact: true })).toBeVisible()
+  await expect(page.getByText('JSONSchema', { exact: true })).toBeVisible()
+})
+
+ftlTest('sends echo.echo request', async ({ page }) => {
+  const sendButton = page.getByText('Send')
+  await sendButton.click()
+
+  await expect(page.getByText('{"message":"Hello')).toBeVisible({ timeout: 30000 })
```
Comment (deniseli) [inline frontend/tests/verb.spec.ts]: Does the timeout need to be this long?

**Review by deniseli — APPROVED** [review]
Yay!!

File: frontend/tests/verb.spec.ts (hunk 1 above)
Comment (wesbillman) [reply frontend/tests/verb.spec.ts]: I'm going to remove that test for now, it's flakey :)

**Comment by wesbillman** [comment]
@alecthomas I feel like maybe this should move to integration tests instead of CI, whatcha think? Do we have a way to add non-go tests to the integration workflow in CI?

**Comment by wesbillman** [comment]
I'm gonna merge this one and follow up with a PR to move these to run in the merge queue and separate out `e2e` tests from normal `test`s.

**Pull request merged**